Bottom: 1cf95f1f38ee1eb47725f46864aebeeb443d8f5f
Top:    e1380093b015e8840315ff5f8087f8827200f77c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 02:54:28 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
